doc/openocd.texi

```diff
@@ -10424,6 +10424,13 @@ tunneled DR scan consists of:
 
 @end deffn
 
+@deffn {Command} {riscv set_maskisr} [@option{off}|@option{steponly}]
+Selects whether interrupts will be disabled when single stepping. The default configuration is @option{off}.
+This feature is only useful on hardware that always steps into interrupts and doesn't support stepie=0.
+Keep in mind, disabling the option does not guarantee that single stepping will go into interrupt handlers.
+To make that happen, stepie would have to be written to 1 as well.
```

Comment:
Just one last cosmetic suggestion: For clarity, it would be good to mention also the register name - `stepie` --> `dcsr.stepie`.

When dealing with RISC-V debug, it is important to be very clear about which registers we talk about, because there are (at least?) three different register spaces involved:

* JTAG registers (IR, DRs)
* Debug module registers (dmstatus, dmcontrol, ...)
* Hart's achitectural registers (GPRs, CSRs, ...)

Thanks.

```suggestion
This feature is only useful on hardware that always steps into interrupts and doesn't support dcsr.stepie=0.
Keep in mind, disabling the option does not guarantee that single stepping will go into interrupt handlers.
To make that happen, dcsr.stepie would have to be written to 1 as well.
```

Reply:
Done

Reply:
Thanks.